bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,31 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
```

Comment:
maybe inline this into the `check_prerequisites` function? I think you don't use it anywhere else + now it looks like some bear-specific-special variable, but it isn't :)